crypto/merkle/tree.go

```diff
@@ -1,22 +1,27 @@
 package merkle
 
 import (
+	"crypto/sha256"
```

Comment:
Isn't this supposed to be `"github.com/minio/sha256-simd"` instead? 
As the changes in this PR make sense independent of simd or not you could either:
 - leave go-changes as is and simply run `go mod tidy` to remove the newly introduced (but not used) dependency, or
 - change the dependency in a central place (tmhash as you suggested)

Reply:
Oh, sorry about this. Just before the PR, I removed the change to test the impact.
I'll do the former as to leave this SIMD change to another pr.